### ISSUE TITLE
WebDriverException: Message: unknown error: Cannot read property 'get' of undefined

### ISSUE THREAD

**Issue by iradovanovic** [rickypc/robotframework-extendedselenium2library#8]
This error message cause javaScript from **init**.py file which is located in NG_WRAPPER.

If you change content of NG_WRAPPER to this:

```
if(window.angular){
var inj;
try{
  inj=angular.element(document.querySelector('[data-ng-app],[ng-app],.ng-cope')||document).injector()
}
catch(ex){
};
if(!inj){
  inj=angular.injector(['ng'])
}
inj.get=inj.get||inj;
inj.get('$browser')
}
```

Then inj won't be anymore undifined. So you just should move (or remove, i don't know all casses)
*\* inj=angular.injector(['ng'])*\*  from catch and put in IF. What should be in catch block - don't know.

(I won't create a MR because i'm not sure about catch)

After this fix, appears new problem and that is only in chrome (In ff everything is working fine with this change), it is about alert window. Tests will be executed like no error but log will show an error:

```
    UnexpectedAlertPresentException: Alert Text: None
Message: unexpected alert open: {Alert text : Are you sure?}
  (Session info: chrome=49.0.2623.87)
  (Driver info: chromedriver=2.21.371459 (36d3d07f660ff2bc1bf28a75d1cdabed0983e7c4),platform=Windows NT 6.3 x86_64)
```

The problem with alert i don't know how to fix it

**Comment by iradovanovic** [comment]
Ok. We found a way how to avoid the problem with the alert. Just use "ALERT SHOULD BE PRESENT      Are you sure? "    instead of Confirm Action. But this still should be fixed.

**Comment by rickypc** [comment]
Thank you, @iradovanovic for reporting the problem. This should be fixed in version 0.8.3. Please reopen otherwise.

In terms of UnexpectedAlertPresentException, I believed that is the correct behavior. I double checked with S2L, ES2L behave the same way as S2L. Please open new issue regarding this if you found otherwise.
